src/ripple/shamap/impl/SHAMap.cpp

```diff
@@ -18,6 +18,7 @@
 //==============================================================================
 
 #include <ripple/basics/contract.h>
+#include <ripple/nodestore/impl/EncodedBlob.h>
```

Comment:
I don't think this `#include` is necessary.

Reply:
removed